tutorials/500_spatiotemporal.ipynb

```diff
@@ -7,23 +7,43 @@
    "source": [
```

Comment:
<p>type in <code>epsilon=0</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>